src/Engine/Renderer/RenderTechnique/ShaderProgram.cpp

```diff
@@ -25,6 +25,59 @@ namespace Engine {
 
 using namespace Core::Utils; // log
 
+// The two following methods are indemendant of any ShaderProgram object. Make them standard local
+// function to remove dependency on openGL.h for the class header
```

Comment:
```suggestion
/// The two following methods are independent of any ShaderProgram object. 
/// \todo Change as local function to remove dependency with openGL.h for the class header
```

Reply:
Well, this is not a todo. This is a Fixed ;)